karma.conf.js

```diff
@@ -50,6 +50,17 @@ const SAUCE_LAUNCHERS = {
 
 const customLaunchers = Object.assign({}, USING_SL ? SAUCE_LAUNCHERS : {}, HEADLESS_LAUNCHERS);
 const browsers = USING_TRAVISCI ? Object.keys(customLaunchers) : ['Chrome'];
+const istanbulInstrumentLoader = {
```

Comment:
Nit: `istanbulInstrumenterLoader`

Seems like we could move `include` into here as well?